posthog/temporal/tests/batch_exports/test_s3_batch_export_workflow.py

```diff
@@ -399,7 +415,11 @@ async def test_insert_into_s3_activity_puts_data_into_s3(
         records_exported = await activity_environment.run(insert_into_s3_activity, insert_inputs)
 
     events_to_export_created, persons_to_export_created = generate_test_data
-    assert records_exported == len(events_to_export_created) or records_exported == len(persons_to_export_created)
+    assert (
+        records_exported == len(events_to_export_created)
+        or records_exported == len(persons_to_export_created)
+        or records_exported == len([event for event in events_to_export_created if event["properties"] is not None])
```

Comment:
also not sure I understand why this is needed (maybe worth adding a comment?)

Reply:
Same reason as above, excluding events without properties.

Reply:
This one I think we'll have to keep, as our tests are hardcoded to export those 5 events without properties. Or we could make that configurable.

Reply:
I have not made that configurable for now. Maybe in a follow-up PR, hoping that's not a blocker.